### DISCUSSION TITLE
Is creating separate accounts per email address preferable?

### DISCUSSION THREAD

**Discussion by adlai** [community/community#48622] [GitHub Education] [answered]
... for avoiding kerfuffles with permissions of [freemium features](./17814), such as those only available to learners.

**Comment by melvincwng** [comment, marked as answer]
Hey @adlai 👋🏻, according to GitHub's Terms of Service over [here](https://docs.github.com/en/site-policy/github-terms/github-terms-of-service#b-account-terms):

> One person or legal entity may maintain no more than one free Account (if you choose to control a machine account as well, that's fine, but it can only be used for running a machine).

Hence, based on that, I **would not** suggest you to create multiple free accounts due to potential violation of the ToS.

Instead, you can consider linking up all of your email address into your main account. For details, you can refer to this GitHub documentation link over [here](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/adding-an-email-address-to-your-github-account) on how to link up multiple email address to your main account.

Hope this helps and answers your question 😃

**Reply by adlai** [reply]
Thank you for the reply; this was my suspicion, thus I did not create an additional account before checking.

My concern is that taking the steps to enable freemium features on the existing account, using student credentials which could expire within one semester, would also end up placing me in violation of the endlessly blooming legalese... although, preventing such situations is **not my job**.
